ci-operator/jobs/RedHatQE/interop-testing/RedHatQE-interop-testing-cnv-4.18-periodics.yaml

```diff
@@ -1,22 +1,23 @@
 periodics:
 - agent: kubernetes
   cluster: build09
-  cron: 04 09 * * 5
+  cron: 0 0 11 6 *
   decorate: true
   decoration_config:
     skip_cloning: true
+    timeout: 6h0m0s
   extra_refs:
   - base_ref: cnv-4.18
     org: RedHatQE
     repo: interop-testing
   labels:
     ci-operator.openshift.io/cloud: aws
-    ci-operator.openshift.io/cloud-cluster-profile: aws-interop-qe
-    ci-operator.openshift.io/variant: ocp-cnv-odf-4.18
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: cnv-odf-ocp4.18-lp-interop
     ci.openshift.io/generator: prowgen
     job-release: "4.18"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-RedHatQE-interop-testing-cnv-4.18-ocp-cnv-odf-4.18-cnv-odf-tests-aws-ipi-ocp418
```

Comment:
@amp-rh - I believe this periodics job yaml is shared by both CNV team and MPIIT. Should their periodic job remain in this file _in addition to_  the `*ocp4.18_lp-interop*` and `*ocp4.18_lp-interop*fips` jobs? (See other periodics.yaml file <4.17.)

Reply:
Good catch! Will fix and push shortly.